spec/features/state_file/complete_intake_spec.rb

```diff
@@ -651,31 +651,40 @@
 
   context "NJ", :flow_explorer_screenshot, js: true do
 
-    def advance_to_start_of_intake(df_persona_name)
+    def advance_to_start_of_intake(df_persona_name, check_a11y: false)
```

Comment:
[dust] I'm wondering why this is a toggle as opposed to always checking accessibility. Is checking costly in terms of time, so turning it off for most tests would reduce test suite time?

Reply:
Yes. I want to avoid duplicate checks of the same pages

Reply:
Makes sense! I'm curious about the scale of time saved, for future reference. But sounds good to me! (Also makes me wonder if there are other places in the slower tests like E2E or PDF filling that we can toggle off to save time... the CI taking 15 minutes is ok but feels like it could be quicker).